<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_window</class>
 <widget class="QMainWindow" name="home_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Início</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 21, 23);
color: rgb(247, 248, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SmartBank</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Olá, Fulano!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="balance_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>123</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Saldo atual</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="balance_value_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>R$ 29.500,00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(6, 150, 255);
border-radius: 16px;
padding: 16px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>461</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QPushButton" name="withdraw_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(31, 33, 36);
	border-radius: 16px;
	padding: 0 8px;	
}

QPushButton:hover {
	background-color: rgb(6, 150, 255);
}
</string>
       </property>
       <property name="text">
        <string>Sacar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deposit_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(31, 33, 36);
	border-radius: 16px;
	padding: 0 8px;	
}

QPushButton:hover {
	background-color: rgb(6, 150, 255);
}
</string>
       </property>
       <property name="text">
        <string>Depositar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="transfer_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(31, 33, 36);
	border-radius: 16px;
	padding: 0 8px;	
}

QPushButton:hover {
	background-color: rgb(6, 150, 255);
}
</string>
       </property>
       <property name="text">
        <string>Transferir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>331</y>
      <width>511</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(31, 33, 36);</string>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="history_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Histórico de transações</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="update_history_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>121</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>48</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 33, 36);
	border-radius: 16px;
	padding: 0 8px;	
}

QPushButton:hover {
	background-color: rgb(31, 33, 36);
}
</string>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QLabel" name="account_code_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Conta: 0000-0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="history_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>49</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>114</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data e hora</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descrição</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Transferência</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>23/10/2000 20:18</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Quantia transferida com sucesso para a conta Nº 1111-1</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>Depósito</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>01/10/2000 20:18</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Quantia depositada com sucesso</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="logout_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>16</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 33, 36);
	border-radius: 16px;
	color: rgb(240, 68, 58);
	padding: 0 8px;	
}

QPushButton:hover {
	background-color: rgb(31, 33, 36);
}
</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label</zorder>
   <zorder>welcome_label</zorder>
   <zorder>balance_label</zorder>
   <zorder>balance_value_label</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>line</zorder>
   <zorder>history_label</zorder>
   <zorder>update_history_btn</zorder>
   <zorder>account_code_label</zorder>
   <zorder>history_table</zorder>
   <zorder>logout_btn</zorder>
  </widget>
 </widget>
 <resources>
  <include location="assets/icons.qrc"/>
 </resources>
 <connections/>
</ui>
